source/slides/05_dns_dhcp.rst

```diff
@@ -268,6 +390,58 @@ resolv.conf has ``nameserver`` entries which tell which dns servers to use::
 
 Most distributions provide a package that manages resolv.conf entries when using dhcp (typically called resolvconf)
 
+Types of DNS servers
+--------------------
+
+**Authoritative**
+
+* Master or Primary
+* Slave or Secondary
+* Hidden Masters
+
+**Non-Authoritative**
+
+* Caching
+* Forwarder
+
+Recursive vs. Non-Recursive
```

Comment:
Let me change this a bit, I was trying to convey which type is typically set as.